integration/tests/cook/test_basic.py

```diff
@@ -1561,6 +1582,10 @@ def check_unscheduled_reason():
         finally:
             util.kill_jobs(self.cook_url, [job_uuid_1, job_uuid_2])
 
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
```

Comment:
Can we have this string in a constant somewhere and refer to the constant in the annotation?

Reply:
Done